lib/cosmos/packets/packet.rb

```diff
@@ -629,7 +629,7 @@ def reset
     #   buffer of data and processors
     def clone
       packet = super()
-      if packet.instance_variable_get("@processors")
+      if packet.instance_variable_get("@processors".freeze)
         packet.instance_variable_set("@processors", packet.processors.clone)
         packet.processors.each do |processor_name, processor|
```

Comment:
Same string constant in next line is not frozen.   Does this really improve performance.   YARV optimizes this somehow?